src/northern_ireland/ni_headcount_fte.py

```diff
@@ -0,0 +1,52 @@
+"""Create the headcount and FTE columns for the NI data."""
+import logging
+import pandas as pd
+import numpy as np
+
+from typing import Dict, List
+
+from src.imputation import apportionment as appt
+
+NIheadcountFTELogger = logging.getLogger(__name__)
+
+
+def create_ni_headcount_cols(df:pd.DataFrame) -> pd.DataFrame:
+    """Create and populate the headcount columns for the NI data."""
+    new_hc_cols = list(appt.hc_dict.keys())
```

Comment:
Would be nice to have a comment explaining this, since it's not immediately obvious what it's doing.